tests/src/test/java/io/streamnative/pulsar/handlers/kop/docker/DockerTest.java

```diff
@@ -28,69 +28,69 @@
 @Slf4j
 public class DockerTest {
 
-    private static final String IMAGE_LUNASTREAMING210 = "datastax/lunastreaming:2.10_3.3";
-    private static final String IMAGE_PULSAR210 = "apachepulsar/pulsar:2.10.3";
+    private static final String IMAGE_LUNASTREAMING31 = "datastax/lunastreaming:3.1";
```

Comment:
this image doens't exist (yet), how can the test work ?

Reply:
We had a discussion over it. We decided to disable tests for this image temporarily. Once that image is available we can enable those tests.